bib/french.bib

```diff
@@ -4946,8 +4946,8 @@ @BOOK(
 )
 
 @MVBOOK(
-	mercenne_1636,
-	author = {F. Marin Mercenne},
+	mersenne_1636,
```

Comment:
Do you consider changind id as really required measure?
This would require adding a redirection in all the four configs (configs/config.*